controller/src/main/java/gov/va/vro/controller/ClaimMetricsController.java

```diff
@@ -74,4 +84,47 @@ public ResponseEntity<List<ExamOrderInfoResponse>> allExamOrderInfo(Integer page
     ExamOrdersInfo examOrdersInfo = claimMetricsService.findAllExamOrderInfo(params);
     return ResponseEntity.ok(examOrdersInfo.getExamOrderInfoList());
   }
+
+  @Override
+  public ResponseEntity<FullHealthDataAssessmentResponse> healthEvidence(
+      HealthDataAssessmentRequest claim) throws ClaimProcessingException {
+    log.info(
+        "Getting full health assessment for claim {} and veteran icn {}",
+        claim.getClaimSubmissionId(),
+        claim.getVeteranIcn());
+    String diagnosis = DiagnosisLookup.getDiagnosis(claim.getDiagnosticCode());
+    if (diagnosis == null) {
+      throw new ClaimProcessingException(
+          claim.getDiagnosticCode(),
+          HttpStatus.BAD_REQUEST,
+          String.format(
+              "Claim with [diagnosticCode = %s] is not in scope.", claim.getDiagnosticCode()));
+    }
+    try {
+      Claim model = postClaimRequestMapper.toModel(claim);
+      model.setIdType(MasAutomatedClaimPayload.CLAIM_V2_ID_TYPE);
```

Comment:
can we repeat the pattern of default to v2, ask for v1? I will confirm with Yoom today about removing the other two endpoints.